### PR TITLE
Update entity group for inventory

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) updates the following article with a new **Inventory Key** field:
https://help.sumologic.com/docs/cse/records-signals-entities-insights/create-an-entity-group/

Issue number: Asana ticket [Document Entity groups enhancement](https://app.asana.com/0/1183741177809740/1205206983224230/f)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/records-signals-entities-insights/create-an-entity-group.md [hunk 1]
```diff
@@ -114,4 +113,4 @@ array_contains(fieldTags["srcDevice_ip"], "DB Server")
 
 ## API support
 
-You can use the CSE `/entity-group-configuration` API to create, read, update and delete Entity Groups. For more information, see [CSE APIs](/docs/cse/administration/cse-apis).
+You can use the the `/entity-group-configuration` API to create, read, update and delete Entity Groups. For more information, see [Cloud SIEM APIs](/docs/cse/administration/cse-apis).
```
Comment (kimsauce) [inline docs/cse/records-signals-entities-insights/create-an-entity-group.md:116]:
```suggestion
You can use the `/entity-group-configuration` API to create, read, update and delete Entity Groups. For more information, see [Cloud SIEM APIs](/docs/cse/administration/cse-apis).
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fix

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
@pkazmir-sumo, could you review this PR?

**Comment by AngelATX** [comment]
Screenshots are a little out of date, might need to update them now. Here is what it looks like now:
Screenshot 2023-08-15 at 4 35 05 PM

Also didn't see much mention of the

**Comment by jpipkin1** [comment]
Thanks for letting me know, @AngelATX. I'll update the screenshots to the latest version available in the CSE Sandbox.

**Review by AngelATX — APPROVED** [review]
Looks good to me.

**Comment by jpipkin1** [comment]
Thanks for the review, @AngelATX.

**Pull request merged**